src/librustdoc/clean/types.rs

```diff
@@ -138,6 +151,8 @@ crate struct ExternalCrate {
 }
 
 impl ExternalCrate {
+    crate const LOCAL: Self = Self { crate_num: LOCAL_CRATE };
```

Comment:
this can be private I think, it's only used from this file
```suggestion
    const LOCAL: Self = Self { crate_num: LOCAL_CRATE };
```